walkthroughs/1-ionic-showcase/walkthrough.adoc

```diff
@@ -25,48 +25,31 @@ image::images/arch.png[architecture, role="integr8ly-img-responsive"]
 * link:{openshift-host}/console/project/{walkthrough-namespace}/overview[OpenShift Project, window="_blank"]
 ****
 
-[type=walkthroughResource]
-.Mobile Developer Console
-****
-* link:{mdc-url}[Mobile Developer Console, window="_blank"]
-****
 
 [type=walkthroughResource]
-.Mobile Developer Services Documentation
-****
-* link:https://access.redhat.com/documentation/en-us/red_hat_managed_integration/1/html-single/getting_started_with_mobile_developer_services/index[Getting Started with Mobile Developer Services, window="_blank"]
-****
-
-[type=walkthroughResource,serviceName=codeready]
-.CodeReady Workspaces
+.{data-sync-name} Documentation
 ****
-* link:{che-url}[Console, window="_blank"]
-* link:https://developers.redhat.com/products/codeready-workspaces/overview/[Overview, window="_blank"]
-* link:https://access.redhat.com/documentation/en-us/red_hat_codeready_workspaces[Documentation, window="_blank"]
+* link:https://github.com/aerogear/voyager-server/tree/master/doc[Getting Started with {data-sync-name}, window="_blank"]
```

Comment:
@pwright are we going to have a link of the data sync docs on customer portal, or it is ok if we joint point to the upstream docs for now?

Reply:
Upstream docs will not exist. I think we agreed with Paul that docs in voyager server will feed directly into access.redhat.com. What it means that we probably might be better with previos link.

@darahayes to confirm